manifests/charts/base/templates/validatingwebhookconfiguration.yaml

```diff
@@ -9,10 +9,14 @@ metadata:
 webhooks:
   - name: validation.istio.io
     clientConfig:
+      {{- if and (.Values.global.remotePilotAddress) (.Values.global.centralIstiod) }}
```

Comment:
Can we stop adding 'pilot' in new configs ? Istiod is the name.

Also I thought we agreed on config going into mesh config and stopping to add global config... 

Why would the address be 'bare' ? All other address ( discoveryAddress, etc) come with port.

And why not use MeshConfig.discoveryAddress directly ?

Reply:
I'm not adding new configs, these are existing configs.

Reply:
this is removed now with validationURL.